packages/amplify-category-custom/src/utils/generate-cfn-from-cdk.ts

```diff
@@ -2,19 +2,32 @@ import * as cdk from 'aws-cdk-lib';
 import { JSONUtilities, pathManager } from '@aws-amplify/amplify-cli-core';
 import * as path from 'path';
 import { categoryName } from './constants';
+import { StackProps } from 'aws-cdk-lib';
 
 export type AmplifyResourceProps = {
   category: string;
   resourceName: string;
 };
 
+export function getCDKProps(): StackProps {
+  if (process.env.CDK_DEFAULT_ACCOUNT || process.env.CDK_DEFAULT_REGION) {
```

Comment:
What is the expected behavior if only one of the envs was set (account but no region and region but no account)?

Reply:
Good question! This environment object is passed to the cdk to build the stack.

According to the cdk docs, if either of those values are not defined, the cdk will substitute it with special strings
https://docs.aws.amazon.com/cdk/v2/guide/environments.html